README.md

```
@@ -304,13 +304,10 @@ Build locally:
 ```bash
 go build -o task ./cmd/task
 ```
-
+I am the original owner of task
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)